docs/get-started/getting-started-kubernetes.md

```diff
@@ -263,5 +263,121 @@ sqlContext.sql("SELECT * FROM df WHERE value<>1").show()
 sc.stop()
 ```
 
+## Running Spark Applications using Spark Operator
+
+Using Spark Operator is another way to submit Spark Applications into a Kubernetes Cluster.
+
+1. Copy Spark Application jars/files into the docker image when preparing docker image.
+
+For example, we copy above `test.py` in directory `/opt/sparkRapidsPlugin` inside the docker image.
+
+2. Create spark operator using `helm`.
+
+Follow [Spark Operator quick start guide](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/quick-start-guide.md)
+
+3. Create a Spark Application YAML file
+
+For example, below file named `testpython-rapids.yaml` is created:
+
+```
+apiVersion: "sparkoperator.k8s.io/v1beta2"
+kind: SparkApplication
+metadata:
+  name: testpython-rapids
+  namespace: default
+spec:
+  sparkConf:
+    "spark.ui.port": "4045"
+    "spark.rapids.sql.concurrentGpuTasks": "1"
+    "spark.executor.resource.gpu.amount": "1"
+    "spark.task.resource.gpu.amount": "1"
```

Comment:
I realize this is done elsewhere in this file and not just here, but do we really want all these examples to setup an executor that can only run 1 task concurrently?

Reply:
I checked all previous examples and all are using 1 task/executor. Since this is for examples, I am fine. 
But if we have this concern, we can point to them to our tuning guide saying something like "For more parameter tuning related topic, refer to this tuning guide." How do you think?

Reply:
It doesn't have to be addressed in this PR, I'm more wondering in general if it's a good idea for these examples to have such a suboptimal setup without calling it out as such.

Reply:
I recalled one reason why we put minimum resource here is sometimes people are testing a single node K8s cluster with 1 GPU inside. This can at least run through without much troubleshooting done to figure out GPU resource not enough issue.

Reply:
1 is the default so we could just leave it off.

Reply:
but I think that's even worse, right?  We don't want only one task running per executor, we would normally want users to set this to a fraction of a GPU.  I'm fine with this going in as-is since it's consistent with the other examples, but I think there should be a followup to address the very suboptimal config especially if the query doesn't run fully on the GPU.

Reply:
Why is that worse?  Our defaults should be something we think are a good starting point, otherwise why is it the default? If you are saying that we should put it in at some other value for most people to start with, we should just change the default and not specify the config explicitly.

Reply:
Oh sorry, I think I was talking bout different configs, I you meant the spark.task.resource.gpu.amount, then agree, setting this lower in example is good.  I was referring to spark.rapids.sql.concurrentGpuTasks

Reply:
I mean it's worse in the sense that it's hidden.  The default makes sense from a safety, it-will-work-if-anything-works perspective, but it's going to be very inefficient for many real-world queries.  My concern is users will set this up per the instructions, get something to run, see the performance is terrible, and stop there.

Like I said above, I'm totally cool with leaving it in (or removing it since it's the default), but we should mention in htis document that the setup is very much untuned/suboptimal and point them to the tuning guide.  The tuning guide needs to discuss setting this config, as it currently doesn't mention it at all.

Reply:
> you meant the spark.task.resource.gpu.amount

Yes, that's what I'm worried about.  Running 1 task per executor and leaving a ton of CPU cores completely unused is not going to be stellar performance for legacy shuffle or other parts of the query that are not GPU-bound.

I'll file a followup issue for addressing this, as the tuning guide really needs to address this and that's outside the scope of this PR.

Reply:
Filed #2328 to track updating the tuning guide and mentioned updating the examples in other docs, so we don't need to deal with that particular issue in this PR.